demisto_sdk/commands/unify/unifier.py

```diff
@@ -137,13 +139,20 @@ def write_yaml_with_docker(self, yml_unified, yml_data, script_obj):
 
         return output_map
 
-    def merge_script_package_to_yml(self):
+    def merge_script_package_to_yml(self, file_name_suffix=None, filter_6_0=False):
         """Merge the various components to create an output yml file
         """
+        if filter_6_0 and parse_version(self.yml_data.get('fromversion', '0.0.0')) >= parse_version('6.0.0'):
```

Comment:
why do we need to check this here and not before the merge too?

Reply:
will move this to creator